index.md

```diff
@@ -4,6 +4,145 @@
 During Operations Rehearsal 5, the scalability of Prompt Processing was limited by the Butler Postgres database.  We propose creating a service to batch the outputs from the individual Prompt Processing pods, reducing contention for the Butler database.  Instead of writing directly to the Butler database, pods would send an event to the service using Kafka.
 ```
 
-## Add content here
+## The problem
 
-See the [Documenteer documentation](https://documenteer.lsst.io/technotes/index.html) for tips on how to write and configure your new technote.
+During Operations Rehearsal 5, the throughput of Prompt Processing was limited
+when Butler Postgres query times increased significantly under high
+concurrency, in some cases taking up to 10 minutes to update the database with
+information about the datasets generated by processing.
+[DM-49293](https://rubinobs.atlassian.net/browse/DM-49293) has additional
+details.
+
+The current Butler queries for inserting dataset records are inefficient for
+the Prompt Processing use case.  They do a large amount of sanity-checking that
+is reasonably efficient when writing many datasets in a batch, but use an
+excessive number of queries and temporary tables when writing a small number of
+datasets of many dataset types.
+
+ However, having hundreds of connections from a large cluster simultaneously
+ writing to a Postgres database may be problematic even if queries are
+ improved.  The Embargo Postgres database used by Prompt Processing is also
+ used by other services and for ad-hoc queries by staff. As the database grows,
+ it will be difficult to guarantee that sufficient resources will be available
+ to meet Prompt Processing time limits.
+
+ Besides the queries that caused problems during the Operations Rehearsal,
+ there are other potential issues lurking.  For example, collection chain
+ updates take a lock that could cause one stalled pod to block completion of
+ all other work.
```

Comment:
This issue will be fixed as part of [DM-45988](https://rubinobs.atlassian.net/browse/DM-45988) -- the init Job is a much more natural place for this operation.

Reply:
Discussed at JTM. @dhirving said that the DM-45988 approach is safer from a provenance standpoint (there would be one true dataset ID).